junebug/workers.py

```diff
@@ -24,8 +30,22 @@ class MessageForwardingWorker(ApplicationWorker):
     CONFIG_CLASS = MessageForwardingConfig
 
     @inlineCallbacks
+    def setup_application(self):
+        self.redis = yield TxRedisManager.from_config(
+            self.config['redis_manager'])
+        message_redis = self.redis.sub_manager(
+            '%s:incoming_messages' % self.config['transport_name'])
```

Comment:
Separate thing -- we need to use the same prefix creating the inbound store on `JunebugApi` for loading messages, so I think we need a place to put the prefix that both the api and this worker can read from. Maybe we should make the prefix be a config field? That way the application worker's can be configured to use the prefix we use in `JunebugApi`.